pom.xml

```diff
@@ -39,6 +39,27 @@
       <scope>provided</scope>
       <optional>true</optional>
     </dependency>
+    <!-- include junit-platform-runner to run JUnit 5 tests with JUnit 4,
+      can be removed after upgrading to maven-surefire-plugin v2.22.0, see SUREFIRE-1330 -->
+    <dependency>
+      <groupId>org.junit.platform</groupId>
+      <artifactId>junit-platform-runner</artifactId>
+      <version>1.2.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <version>5.2.0</version>
+      <scope>provided</scope>
+      <optional>true</optional>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <version>5.2.0</version>
```

Comment:
should junit-jupiter-api and junit-jupiter-engine be in sync ? if so let's extract the version to a property

Reply:
Yes, they should be in sync, also the junit-platform-runner. Either we extract that as property or we use the [BOM](https://junit.org/junit5/docs/current/user-guide/#dependency-metadata-junit-bom) `org.junit:junit-bom` which is available since 5.2.0.

Should this be part of assertj-core or of assertj- parent-pom?

Reply:
Let's use the BOM (good idea) and keep it in assertj-core at the moment.